flexmeasures/ui/templates/defaults.jinja

```diff
@@ -20,12 +20,15 @@
 %}
 
 {% for view_name in FLEXMEASURES_LISTED_VIEWS %}
-    {# add specs for views we don't know (plugin views) #}
-    {% do nav_bar_specs.update({view_name: dict(title=view_name.capitalize(), tooltip="", icon="info")}) if view_name not in nav_bar_specs %}
-    {# add view to menu if user is authenticated #}
-    {% do navigation_bar.append(
-        (view_name, view_name, nav_bar_specs[view_name]["title"], nav_bar_specs[view_name]["tooltip"], nav_bar_specs[view_name]["icon"])
-    ) if current_user.is_authenticated %}
+    {% set view_name = view_name | parsed_view_config_item("FLEXMEASURES_LISTED_VIEWS") %}
```

Comment:
Interesting, I didn't know you could pass two arguments to a Jinja filter like this. I learned something, thanks.

Reply:
Well, it did take an hour of my time or so to get this part right, as well :/